server/api/discrepancies.py

```diff
@@ -195,32 +229,61 @@ def get_ballot_comparison_audit_discrepancies_by_jurisdiction(
         reported_results = cvrs_for_contest(contest)
         for ballot_id, audited_result in audited_results.items():
-            audited_cvr = audited_result["cvr"]
-            reported_cvr = reported_results.get(ballot_id)
-            vote_deltas = ballot_vote_deltas(contest, reported_cvr, audited_cvr)
-            if not vote_deltas or isinstance(vote_deltas, str):
-                continue
-
-            # CVRs are guaranteed to be non-null due to ballot_vote_deltas() checks
-            assert isinstance(audited_cvr, dict)
-            audited_votes = audited_cvr.get(contest.id, {})
-            assert isinstance(reported_cvr, dict)
-            reported_votes = reported_cvr.get(contest.id, {})
             if ballot_id in sampled_ballot_id_to_jurisdiction_id:
                 jurisdiction_id = sampled_ballot_id_to_jurisdiction_id[ballot_id]
-                readable_ballot_identifier = sampled_ballot_id_to_readable_identifier[
-                    ballot_id
-                ]
-                discrepancies_by_jurisdiction[jurisdiction_id][
-                    readable_ballot_identifier
-                ][contest.id] = {
+                if not show_discrepancies_by_jurisdiction[jurisdiction_id]:
+                    continue
+
+                audited_cvr = audited_result["cvr"]
+                reported_cvr = reported_results.get(ballot_id)
+                vote_deltas = ballot_vote_deltas(contest, reported_cvr, audited_cvr)
+                if not vote_deltas or isinstance(vote_deltas, str):
+                    continue
+
+                # CVRs are guaranteed to be non-null due to ballot_vote_deltas() checks
+                assert isinstance(audited_cvr, dict)
+                audited_votes = audited_cvr.get(contest.id, {})
+                assert isinstance(reported_cvr, dict)
+                reported_votes = reported_cvr.get(contest.id, {})
+
+                readable_ballot_id = sampled_ballot_id_to_readable_id[ballot_id]
+                discrepancies_by_jurisdiction[jurisdiction_id][readable_ballot_id][
+                    contest.id
+                ] = {
                     "reportedVotes": reported_votes,
                     "auditedVotes": audited_votes,
                     "discrepancies": vote_deltas,
                 }
 
     return discrepancies_by_jurisdiction
+
+
+# In multi-jurisdiction audits, hide discrepancies if the jurisdiction hasn't signed off
+def show_ballot_comparison_discrepancies_by_jurisdiction(
+    election: Election,
+) -> Dict[str, bool]:
+    jurisdictions = list(election.jurisdictions)
+    is_single_jurisdiction_election = len(jurisdictions) == 1
+    if is_single_jurisdiction_election:
+        jurisidiction_id = jurisdictions[0].id
+        return {jurisidiction_id: True}
+
+    rounds = list(election.rounds)
+    audit_board_status_by_jurisdiction = jurisdiction_audit_board_status(
+        jurisdictions, rounds[-1]
```

Comment:
Convention is to use `get_current_round` rather than manipulate the rounds list manually